pkg/stub/handler.go

```diff
@@ -812,6 +825,22 @@ func (h *Handler) ProcessManagementField(srcfg *v1alpha1.SamplesResource) (bool,
 	case operatorsv1api.Managed:
 		if srcfg.Spec.ManagementState != srcfg.Status.ManagementState {
 			logrus.Println("management state set to managed")
+			if srcfg.Spec.InstallType == v1alpha1.RHELSamplesDistribution &&
+				srcfg.ConditionFalse(v1alpha1.ImportCredentialsExist) {
+				secret, err := h.secretclientwrapper.Get(h.namespace, v1alpha1.SamplesRegistryCredentials)
+				if err == nil && secret != nil {
+					// initiate a secret event vs. doing the copy ourselves
+					go func() {
+						logrus.Println("updating operator namespace credential to initiate creation of credential in openshift namespace")
+						if secret.Annotations == nil {
+							secret.Annotations = map[string]string{}
+						}
+						// testing showed that we needed to change something for the update to actually go through
+						secret.Annotations[v1alpha1.SamplesRecreateCredentialAnnotation] = kapis.Now().String()
+						h.secretclientwrapper.Update(h.namespace, secret)
+					}()
```

Comment:
why does this need to be done in a go routine?

Reply:
it doesn't *HAVE* to be in a go routine, but there is not reason to gate returning to the sdk with this poking of the secret ... it is just a slight performance improvement

also I am augmenting the comment here as well with more details on why we are doing this

but if you really want it out of the go routine just let me know and I'll change it

Reply:
Yeah my inclination would be to drop the go routine in favor of less complexity and more deterministic flows.